insonmnia/hardware/marshal.go

```diff
@@ -120,13 +121,19 @@ func GPUFromProto(g []*pb.GPUDevice) ([]*gpu.Device, error) {
 			cacheSize = 0
 		}
 
+		maxMemorySize, err := strconv.ParseUint(i.Ext["max_memory_size"], 10, 0)
+		if err != nil {
+			cacheSize = 0
```

Comment:
Why `cacheSize = 0` if you cannot parse `maxMemorySize`?

Reply:
Wow, typo.

Reply:
Fixed. Anyway I want to migrate to strictly-typed messages when we stabilize what we exactly want from devices.